<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_teacher</class>
 <widget class="QDialog" name="Add_teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="labelSpisokPrepodvatelei">
       <property name="text">
        <string>Список преподавателей</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetPrepodavatelei"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelAddPrepodavatlei">
       <property name="text">
        <string>Добавить преподавателя</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditAddPrepodavatelei"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonCancel">
           <property name="text">
            <string>Отменить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDelete">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
